en/docs/integrate/examples/working-with-transactions.md

```diff
@@ -174,6 +174,26 @@ The sample scenario can be depicted as follows:
     ```xml
     <property name="SET_ROLLBACK_ONLY" scope="default" type="STRING" value="true"/>
     ```
+
+    You can alterntively use Client Acknowledgment of JMS (this will not slow down message consumption)
+
+    Configure JMS transport or inbound JMS protocol to use JMS Client Acknowledgment:
+
+    ```  
+    <parameter name="transport.jms.CacheLevel">consumer</parameter>
+    <parameter name="transport.jms.SessionAcknowledgement">CLIENT_ACKNOWLEDGE</parameter>
```

Comment:
We should not configure these configs in a transacted session. We need to set the following property as well, `<parameter name="transport.jms.SessionTransacted">false</parameter>`. IMO, we can move this out of Transactions under new heading.

Reply:
@Vathsan do you mean a different section in this same page or a completely different page? What would we call it?

Reply:
It is better if we can have a new page, something like working-with-client-acknowledgement. But I suppose we do not have enough content to create a new page. I think we can have this as a note in this page. WDYT?

Reply:
Ok, I'll add it as a note.